Sources/Core/Sources/Render/RenderConfiguration.swift

```diff
@@ -13,14 +13,19 @@ public struct RenderConfiguration: Codable {
   /// The rendering mode to use.
   public var mode: RenderMode
   
+  /// Defines upscaling factor (if supported).
+  ///  Defaults to 0 (no upscaling)
+  public var upscaleFactor: Int
+  
   /// Creates a new render configuration.
   /// - Parameters:
   ///   - fovY: See ``fovY``. Defaults to 70 degrees.
   ///   - renderDistance: See ``renderDistance``. Defaults to 10.
   ///   - mode: See ``mode``. Defaults to ``RenderMode/normal``.
```

Comment:
Document the new parameter (easy to forget lol, I do it all the time)

Reply:
True, true